go/engine/engine.go

```diff
@@ -61,6 +77,21 @@ func runPrereqs(e Engine, ctx *Context) (err error) {
 
 }
 
+func tryStandaloneLogin(e Engine, ctx *Context) error {
+	if !e.G().Env.GetStandalone() {
```

Comment:
this doesn't seem right.  won't this be producing errors in the case of client-service mode?